### PR TITLE
[KOGITO-6229] - Add Oracle DB Support to Data Index

### PR DESCRIPTION
**Pull request by Scuilion**
See: https://issues.redhat.com/browse/KOGITO-6229

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by spolti** [comment]
Hi @Scuilion, thanks very much for your contribution.
Can you please update the commit's message to ```[KOGITO-6229] - Add Oracle DB Support to Data Index```?

Thanks.

File: README.md [hunk 1]
```diff
@@ -582,6 +582,8 @@ The Persistence service can be switched by using its corresponding image
   [image.yaml](kogito-data-index-ephemeral-overrides.yaml)
 - Mongodb: quay.io/kiegroup/kogito-data-index-mongodb 
   [image.yaml](kogito-data-index-mongodb-overrides.yaml)
+- Oracle: quay.io/kiegroup/kogito-data-index-Oracle
```
Comment (spolti) [inline README.md]: use lower case.

**Comment by spolti** [comment]
I don't think there is a oracle extension for data-index at this point. @cristianonicolai, can you please confirm?

**Comment by Scuilion** [comment]
@spolti There is a PR currently in review for this:
https://github.com/kiegroup/kogito-apps/pull/1159

**Comment by radtriste** [comment]
https://github.com/kiegroup/kogito-apps/pull/1159 has been merged just in time before the nightly run. 

Artifacts is available: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-service-oracle/2.0.0-SNAPSHOT/data-index-service-oracle-2.0.0-20220120.051951-1-runner.jar
[md5](https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-service-oracle/2.0.0-SNAPSHOT/data-index-service-oracle-2.0.0-20220120.051951-1-runner.jar.md5): 6639b8d22b2843130bb8d42cc284729e

**Review by spolti — CHANGES_REQUESTED** [review]
can you please update scripts/update-maven-artifacts.py scripts as well?

**Review by cristianonicolai — APPROVED** [review]

File: modules/kogito-data-index-oracle/module.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+schema_version: 1
+name: org.kie.kogito.dataindex.oracle
+version: "2.0.0-snapshot"
+
+artifacts:
+  - name: data-index-service-oracle-runner.jar
+    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-service-oracle/2.0.0-SNAPSHOT/data-index-service-oracle-2.0.0-SNAPSHOT-runner.jar
```
Comment (radtriste) [inline modules/kogito-data-index-oracle/module.yaml]: Please use a specific snapshot version here.

```suggestion
    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-service-oracle/2.0.0-SNAPSHOT/data-index-service-oracle-2.0.0-20220124.043203-5-runner.jar
```

**Comment by radtriste** [comment]
jenkins retest this

**Comment by kie-ci1** [comment]
**PR job** #185 was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/185/display/redirect or see console log:
```spoiler Console Logs
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Prepare offline kogito-examples)
Stage "Prepare offline kogito-examples" skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Build & Test Images)
Stage "Build & Test Images" skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Declarative: Post Actions)
[Pipeline] script
[Pipeline] {
[Pipeline] sh
+ rm -rf /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-images/workspaces
[Pipeline] sh
++ docker ps -a -q
+ docker rm -f
"docker rm" requires at least 1 argument.
See 'docker rm --help'.

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers
+ date
Mon Jan 24 03:42:32 EST 2022
[Pipeline] sh
++ docker images -q
+ docker rmi -f
"docker rmi" requires at least 1 argument.
See 'docker rmi --help'.

Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]

Remove one or more images
+ date
Mon Jan 24 03:42:33 EST 2022
[Pipeline] sh
+ rm -rf /home/jenkins/.cekit/cache
[Pipeline] }
[Pipeline] // script
[Pipeline] script
[Pipeline] {
[Pipeline] sh
+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/185/api/json
[Pipeline] readJSON
[Pipeline] sh

```

**Comment by radtriste** [comment]
```
-------------------------------------------------------------

        [ERROR] Can't merge source into Target. Please rebase PR branch.

        -------------------------------------------------------------

        Source: git://github.com/Scuilion/kogito-images KOGITO-5921-example-app-jdbc-oracle

        Target: 08d0dbdc BXMSPROD-1579 Send build summary as PR comment (#1043)

        -------------------------------------------------------------
```
@Scuilion can you please rebase your branch ?

**Comment by radtriste** [comment]
jenkins retest this

**Comment by kie-ci1** [comment]
**PR job** #187 was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Build failures on those images:
- kogito-data-index-oracle



**Test results:**
- PASSED: 235
- FAILED: 0

Those are the test failures: none

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/187/display/redirect or see console log:
```spoiler kogito-data-index-oracle
[Pipeline] sh
[32m2022-01-24 10:33:48,019 docker_builder.py:108        INFO  Docker: Installing: nss-softokn-freebl;3.67.0-7.el8_5;x86_64;ubi-8-appstream[0m
[32m2022-01-24 10:33:47,941 docker_builder.py:108        INFO  Docker: Downloading metadata...[0m
+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/187/consoleText
+ tail -n 100
[32m2022-01-24 10:33:48,058 docker_builder.py:108        INFO  Docker: Package                                                          Repository          Size[0m
[32m2022-01-24 10:33:48,058 docker_builder.py:108        INFO  Docker: Installing:                                                                              [0m
[32m2022-01-24 10:33:48,059 docker_builder.py:108        INFO  Docker:  alsa-lib-1.2.5-4.el8.x86_64                                     ubi-8-appstream 501.1 kB[0m
[32m2022-01-24 10:33:48,059 docker_builder.py:108        INFO  Docker:  avahi-libs-0.7-20.el8.x86_64                                    ubi-8-baseos     63.1 kB[0m
[32m2022-01-24 10:33:48,059 docker_builder.py:108        INFO  Docker:  copy-jdk-configs-4.0-2.el8.noarch                               ubi-8-appstream  31.3 kB[0m
[32m2022-01-24 10:33:48,059 docker_builder.py:108        INFO  Docker:  crypto-policies-scripts-20210617-1.gitc776d3e.el8.noarch        ubi-8-baseos     84.8 kB[0m
[32m2022-01-24 10:33:48,059 docker_builder.py:108        INFO  Docker:  cups-libs-1:2.2.6-40.el8.x86_64                                 ubi-8-baseos    443.6 kB[0m
[32m2022-01-24 10:33:48,060 docker_builder.py:108        INFO  Docker:  dbus-libs-1:1.12.8-14.el8.x86_64                                ubi-8-baseos    188.0 kB[0m
[32m2022-01-24 10:33:48,060 docker_builder.py:108        INFO  Docker:  expat-2.2.5-4.el8.x86_64                                        ubi-8-baseos    113.6 kB[0m
[32m2022-01-24 10:33:48,060 docker_builder.py:108        INFO  Docker:  findutils-1:4.6.0-20.el8.x86_64                                 ubi-8-baseos    540.8 kB[0m
[32m2022-01-24 10:33:48,060 docker_builder.py:108        INFO  Docker:  freetype-2.9.1-4.el8_3.1.x86_64                                 ubi-8-baseos    403.1 kB[0m
[32m2022-01-24 10:33:48,060 docker_builder.py:108        INFO  Docker:  gdbm-1:1.18-1.el8.x86_64                                        ubi-8-baseos    132.9 kB[0m
[32m2022-01-24 10:33:48,060 docker_builder.py:108        INFO  Docker:  gdbm-libs-1:1.18-1.el8.x86_64                                   ubi-8-baseos     62.0 kB[0m
[32m2022-01-24 10:33:48,060 docker_builder.py:108        INFO  Docker:  graphite2-1.3.10-10.el8.x86_64                                  ubi-8-appstream 124.5 kB[0m
[32m2022-01-24 10:33:48,060 docker_builder.py:108        INFO  Docker:  harfbuzz-1.7.5-3.el8.x86_64                                     ubi-8-appstream 301.2 kB[0m
[32m2022-01-24 10:33:48,060 docker_builder.py:108        INFO  Docker:  java-11-openjdk-headless-1:11.0.14.0.9-2.el8_5.x86_64           ubi-8-appstream  41.5 MB[0m
[32m2022-01-24 10:33:48,061 docker_builder.py:108        INFO  Docker:  javapackages-filesystem-5.3.0-1.module+el8+2447+6f56d9a6.noarch ubi-8-appstream  31.1 kB[0m
[32m2022-01-24 10:33:48,061 docker_builder.py:108        INFO  Docker:  lcms2-2.9-2.el8.x86_64                                          ubi-8-appstream 168.6 kB[0m
[32m2022-01-24 10:33:48,061 docker_builder.py:108        INFO  Docker:  libjpeg-turbo-1.5.3-12.el8.x86_64                               ubi-8-appstream 160.9 kB[0m
[32m2022-01-24 10:33:48,061 docker_builder.py:108        INFO  Docker:  libnsl2-1.2.0-2.20180605git4a062cf.el8.x86_64                   ubi-8-baseos     59.1 kB[0m
[32m2022-01-24 10:33:48,061 docker_builder.py:108        INFO  Docker:  libpng-2:1.6.34-5.el8.x86_64                                    ubi-8-baseos    129.0 kB[0m
[32m2022-01-24 10:33:48,061 docker_builder.py:108        INFO  Docker:  libtirpc-1.1.4-5.el8.x86_64                                     ubi-8-baseos    114.7 kB[0m
[32m2022-01-24 10:33:48,061 docker_builder.py:108        INFO  Docker:  lksctp-tools-1.0.18-3.el8.x86_64                                ubi-8-baseos    101.9 kB[0m
[32m2022-01-24 10:33:48,061 docker_builder.py:108        INFO  Docker:  lua-5.3.4-12.el8.x86_64                                         ubi-8-appstream 197.0 kB[0m
[32m2022-01-24 10:33:48,064 docker_builder.py:108        INFO  Docker:  nspr-4.32.0-1.el8_4.x86_64                                      ubi-8-appstream 145.9 kB[0m
[32m2022-01-24 10:33:48,064 docker_builder.py:108        INFO  Docker:  nss-3.67.0-7.el8_5.x86_64                                       ubi-8-appstream 758.6 kB[0m
[32m2022-01-24 10:33:48,064 docker_builder.py:108        INFO  Docker:  nss-softokn-3.67.0-7.el8_5.x86_64                               ubi-8-appstream 498.4 kB[0m
[32m2022-01-24 10:33:48,064 docker_builder.py:108        INFO  Docker:  nss-softokn-freebl-3.67.0-7.el8_5.x86_64                        ubi-8-appstream 404.7 kB[0m
[32m2022-01-24 10:33:48,065 docker_builder.py:108        INFO  Docker:  nss-sysinit-3.67.0-7.el8_5.x86_64                               ubi-8-appstream  74.9 kB[0m
[32m2022-01-24 10:33:48,065 docker_builder.py:108        INFO  Docker:  nss-util-3.67.0-7.el8_5.x86_64                                  ubi-8-appstream 140.4 kB[0m
[32m2022-01-24 10:33:48,065 docker_builder.py:108        INFO  Docker:  platform-python-3.6.8-41.el8.x86_64                             ubi-8-baseos     86.9 kB[0m
[32m2022-01-24 10:33:48,065 docker_builder.py:108        INFO  Docker:  platform-python-setuptools-39.2.0-6.el8.noarch                  ubi-8-baseos    647.5 kB[0m
[32m2022-01-24 10:33:48,065 docker_builder.py:108        INFO  Docker:  python3-libs-3.6.8-41.el8.x86_64                                ubi-8-baseos      8.2 MB[0m
[32m2022-01-24 10:33:48,065 docker_builder.py:108        INFO  Docker:  python3-pip-wheel-9.0.3-20.el8.noarch                           ubi-8-baseos      1.1 MB[0m
[32m2022-01-24 10:33:48,065 docker_builder.py:108        INFO  Docker:  python3-setuptools-wheel-39.2.0-6.el8.noarch                    ubi-8-baseos    295.8 kB[0m
[32m2022-01-24 10:33:48,065 docker_builder.py:108        INFO  Docker:  tzdata-java-2021e-1.el8.noarch                                  ubi-8-appstream 195.7 kB[0m
[32m2022-01-24 10:33:48,066 docker_builder.py:108        INFO  Docker: Transaction Summary:[0m
[32m2022-01-24 10:33:48,066 docker_builder.py:108        INFO  Docker:  Installing:       34 packages[0m
[32m2022-01-24 10:33:48,066 docker_builder.py:108        INFO  Docker:  Reinstalling:      0 packages[0m
[32m2022-01-24 10:33:48,066 docker_builder.py:108        INFO  Docker:  Upgrading:         0 packages[0m
[32m2022-01-24 10:33:48,066 docker_builder.py:108        INFO  Docker:  Obsoleting:        0 packages[0m
[32m2022-01-24 10:33:48,066 docker_builder.py:108        INFO  Docker:  Removing:          0 packages[0m
[32m2022-01-24 10:33:48,067 docker_builder.py:108        INFO  Docker:  Downgrading:       0 packages[0m
[32m2022-01-24 10:33:48,067 docker_builder.py:108        INFO  Docker: Enabling module streams:[0m
[32m2022-01-24 10:33:48,067 docker_builder.py:108        INFO  Docker:     javapackages-runtime:201801[0m
[32m2022-01-24 10:33:48,067 docker_builder.py:108        INFO  Docker: Downloading packages...[0m
[32m2022-01-24 10:33:48,316 docker_builder.py:108        INFO  Docker: Installing: nss-softokn;3.67.0-7.el8_5;x86_64;ubi-8-appstream[0m
[32m2022-01-24 10:33:48,209 docker_builder.py:108        INFO  Docker: Installing: tzdata-java;2021e-1.el8;noarch;ubi-8-appstream[0m
[32m2022-01-24 10:33:48,700 docker_builder.py:108        INFO  Docker: Installing: lua;5.3.4-12.el8;x86_64;ubi-8-appstream[0m
[32m2022-01-24 10:33:48,820 docker_builder.py:108        INFO  Docker: Package                                                          Repository          Size[0m
[32m2022-01-24 10:33:48,821 docker_builder.py:108        INFO  Docker: Installing:                                                                              [0m
[32m2022-01-24 10:33:48,821 docker_builder.py:108        INFO  Docker:  alsa-lib-1.2.5-4.el8.x86_64                                     ubi-8-appstream 501.1 kB[0m
[32m2022-01-24 10:33:48,821 docker_builder.py:108        INFO  Docker:  avahi-libs-0.7-20.el8.x86_64                                    ubi-8-baseos     63.1 kB[0m
[32m2022-01-24 10:33:48,821 docker_builder.py:108        INFO  Docker:  copy-jdk-configs-4.0-2.el8.noarch                               ubi-8-appstream  31.3 kB[0m
[32m2022-01-24 10:33:48,821 docker_builder.py:108        INFO  Docker:  crypto-policies-scripts-20210617-1.gitc776d3e.el8.noarch        ubi-8-baseos     84.8 kB[0m
[32m2022-01-24 10:33:48,821 docker_builder.py:108        INFO  Docker:  cups-libs-1:2.2.6-40.el8.x86_64                                 ubi-8-baseos    443.6 kB[0m
[32m2022-01-24 10:33:48,821 docker_builder.py:108        INFO  Docker:  dbus-libs-1:1.12.8-14.el8.x86_64                                ubi-8-baseos    188.0 kB[0m
[32m2022-01-24 10:33:48,821 docker_builder.py:108        INFO  Docker:  expat-2.2.5-4.el8.x86_64                                        ubi-8-baseos    113.6 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  findutils-1:4.6.0-20.el8.x86_64                                 ubi-8-baseos    540.8 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  freetype-2.9.1-4.el8_3.1.x86_64                                 ubi-8-baseos    403.1 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  gdbm-1:1.18-1.el8.x86_64                                        ubi-8-baseos    132.9 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  gdbm-libs-1:1.18-1.el8.x86_64                                   ubi-8-baseos     62.0 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  graphite2-1.3.10-10.el8.x86_64                                  ubi-8-appstream 124.5 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  harfbuzz-1.7.5-3.el8.x86_64                                     ubi-8-appstream 301.2 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  java-11-openjdk-headless-1:11.0.14.0.9-2.el8_5.x86_64           ubi-8-appstream  41.5 MB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  javapackages-filesystem-5.3.0-1.module+el8+2447+6f56d9a6.noarch ubi-8-appstream  31.1 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  lcms2-2.9-2.el8.x86_64                                          ubi-8-appstream 168.6 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  libjpeg-turbo-1.5.3-12.el8.x86_64                               ubi-8-appstream 160.9 kB[0m
[32m2022-01-24 10:33:48,822 docker_builder.py:108        INFO  Docker:  libnsl2-1.2.0-2.20180605git4a062cf.el8.x86_64                   ubi-8-baseos     59.1 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  libpng-2:1.6.34-5.el8.x86_64                                    ubi-8-baseos    129.0 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  libtirpc-1.1.4-5.el8.x86_64                                     ubi-8-baseos    114.7 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  lksctp-tools-1.0.18-3.el8.x86_64                                ubi-8-baseos    101.9 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  lua-5.3.4-12.el8.x86_64                                         ubi-8-appstream 197.0 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  nspr-4.32.0-1.el8_4.x86_64                                      ubi-8-appstream 145.9 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  nss-3.67.0-7.el8_5.x86_64                                       ubi-8-appstream 758.6 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  nss-softokn-3.67.0-7.el8_5.x86_64                               ubi-8-appstream 498.4 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  nss-softokn-freebl-3.67.0-7.el8_5.x86_64                        ubi-8-appstream 404.7 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  nss-sysinit-3.67.0-7.el8_5.x86_64                               ubi-8-appstream  74.9 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  nss-util-3.67.0-7.el8_5.x86_64                                  ubi-8-appstream 140.4 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  platform-python-3.6.8-41.el8.x86_64                             ubi-8-baseos     86.9 kB[0m
[32m2022-01-24 10:33:48,823 docker_builder.py:108        INFO  Docker:  platform-python-setuptools-39.2.0-6.el8.noarch                  ubi-8-baseos    647.5 kB[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  python3-libs-3.6.8-41.el8.x86_64                                ubi-8-baseos      8.2 MB[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  python3-pip-wheel-9.0.3-20.el8.noarch                           ubi-8-baseos      1.1 MB[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  python3-setuptools-wheel-39.2.0-6.el8.noarch                    ubi-8-baseos    295.8 kB[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  tzdata-java-2021e-1.el8.noarch                                  ubi-8-appstream 195.7 kB[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker: Transaction Summary:[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  Installing:       34 packages[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  Reinstalling:      0 packages[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  Upgrading:         0 packages[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  Obsoleting:        0 packages[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  Removing:          0 packages[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker:  Downgrading:       0 packages[0m
[32m2022-01-24 10:33:48,824 docker_builder.py:108        INFO  Docker: Enabling module streams:[0m
[32m2022-01-24 10:33:48,825 docker_builder.py:108        INFO  Docker:     javapackages-runtime:201801[0m
[32m2022-01-24 10:33:48,828 docker_builder.py:108        INFO  Docker: Downloading packages...[0m

```

File: modules/kogito-data-index-oracle/module.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+schema_version: 1
+name: org.kie.kogito.dataindex.oracle
+version: "2.0.0-snapshot"
+
+artifacts:
+  - name: data-index-service-oracle-runner.jar
+    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-service-oracle/2.0.0-SNAPSHOT/data-index-service-oracle-2.0.0-20220124.043203-5-runner.jar
+    md5: 1ccd9f7835e0b23d7d7d3c1aa50e6021
```
Comment (radtriste) [inline modules/kogito-data-index-oracle/module.yaml]:
```suggestion
    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-service-oracle/2.0.0-SNAPSHOT/data-index-service-oracle-2.0.0-20220124.043203-5-runner.jar
    md5: 9dd485c017dc9e61efaadc27fcf4c0ba
```

Comment (radtriste) [reply modules/kogito-data-index-oracle/module.yaml]: sorry forgot to update the md5 of the runner ...

**Comment by radtriste** [comment]
jenkins retest this

**Review by spolti — APPROVED** [review]

**Comment by kie-ci1** [comment]
**PR job** #188 was: **UNSTABLE**
Possible explanation: This should be test failures


**Test results:**
- PASSED: 237
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/188/testReport/features.image.data-index.kogito-data-index-oracle.Kogito-data-index oracle feature//verify_if_of_container_is_correctly_started_with_oracle_parameters/">features.image.data-index.kogito-data-index-oracle.Kogito-data-index oracle feature..verify if of container is correctly started with oracle parameters</a></summary>
Message 'Datasource '<default>': The connection attempt failed' was not found in the logs
</details>

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/188/display/redirect

**Comment by radtriste** [comment]
@Scuilion looks like the JDBC URL in test is wrong, it should be something like `jdbc:oracle:thin:@10.1.1.10:5432/quarkus`
Also, there should be a connection timeout for the test to fail quickly, else it does try to connect for quite some time.

File: tests/features/data-index/kogito-data-index-oracle.feature [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+  Scenario: verify if the indexing service binaries are available on /home/kogito/bin
+    When container is started with command bash
+    Then run sh -c 'ls /home/kogito/bin/data-index-service-oracle-runner.jar' in container and immediately check its output for /home/kogito/bin/data-index-service-oracle-runner.jar
+
+  Scenario: verify if of container is correctly started with oracle parameters
+    When container is started with env
+      | variable                     | value                                     |
+      | SCRIPT_DEBUG                 | true                                      |
+      | QUARKUS_DATASOURCE_JDBC_URL  | jdbc:oracle:thin://10.1.1.10:5432/quarkus |
```
Comment (spolti) [inline tests/features/data-index/kogito-data-index-oracle.feature]:
```suggestion
      | QUARKUS_DATASOURCE_JDBC_URL  | jdbc:oracle:thin:@//10.1.1.53:1521/quarkus |
```

File: tests/features/data-index/kogito-data-index-oracle.feature [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      | QUARKUS_DATASOURCE_USERNAME  | kogito                                    |
+      | QUARKUS_DATASOURCE_PASSWORD  | s3cr3t                                    |
+    Then container log should contain -Djava.library.path=/home/kogito/lib -Dquarkus.http.host=0.0.0.0 -Dquarkus.http.port=8080 -jar /home/kogito/bin/data-index-service-oracle-runner.jar
+    And container log should contain Datasource '<default>': The connection attempt failed
```
Comment (spolti) [inline tests/features/data-index/kogito-data-index-oracle.feature]:
```suggestion
    And container log should contain Datasource '<default>': IO Error: The Network Adapter could not establish the connection
```

**Comment by kie-ci1** [comment]
**PR job** #194 was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/194/display/redirect or see console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Prepare offline kogito-examples)<br/>Stage "Prepare offline kogito-examples" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-images/workspaces<br/>[Pipeline] sh<br/>++ docker ps -a -q<br/>+ docker rm -f<br/>"docker rm" requires at least 1 argument.<br/>See 'docker rm --help'.<br/><br/>Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]<br/><br/>Remove one or more containers<br/>+ date<br/>Thu Feb  3 12:03:51 EST 2022<br/>[Pipeline] sh<br/>++ docker images -q<br/>+ docker rmi -f<br/>"docker rmi" requires at least 1 argument.<br/>See 'docker rmi --help'.<br/><br/>Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]<br/><br/>Remove one or more images<br/>+ date<br/>Thu Feb  3 12:03:51 EST 2022<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.cekit/cache<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/194/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by spolti** [comment]
@radtriste is there something we could do to address the git issue on the ci?

**Comment by radtriste** [comment]
well... a rebase is needed ...

**Comment by spolti** [comment]
thanks @radtriste @Scuilion 
/jenkins retest this

**Comment by kie-ci1** [comment]
**PR job** `#195` was: **UNSTABLE**
Possible explanation: This should be test failures


**Test results:**
- PASSED: 237
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/195/testReport/features.image.kogito-builder/kogito-builder image tests/Verify_that_the_Kogito_Quarkus_Serverless_Workflow_Extension_is_building_the_service_properly/">features.image.kogito-builder.kogito-builder image tests.Verify that the Kogito Quarkus Serverless Workflow Extension is building the service properly</a></summary>
S2I build failed, check logs!
</details>

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/195/display/redirect

**Comment by radtriste** [comment]
Issue coming from artifacts snapshots not updated on Nexus ...

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
cherry-pick to 1.17.x: https://github.com/kiegroup/kogito-images/pull/1076

**Pull request merged**